db/migrate/20150120095012_create_categories.rb

```diff
@@ -2,6 +2,7 @@ class CreateCategories < ActiveRecord::Migration
   def change
     create_table :categories do |t|
       t.string :name
+      t.string :link
```

Comment:
Actually I dont think I understand this part either. but scrap away, will clarify with you in person when you're back